nilearn/tests/test_signal.py

```diff
@@ -366,7 +367,23 @@ def test_clean_confounds():
                                    np.zeros((20, 2)))
 
 
+def test_clean_img():
+
```

Comment:
You should create a local rng, and not sample from the global one, using something like:

rng = np.random.RandomState(0)